docs/tutorials/intro.md

```diff
@@ -182,7 +154,7 @@ and then we can use it in our code:
 use icu_provider_fs::FsDataProvider;
 
 fn main() {
-    let _provider = FsDataProvider::try_new("/home/{USER}/projects/icu/icu4x-data")
+    let _provider = FsDataProvider::try_new("/path/to/icu4x/provider/testdata/data/json")
```

Comment:
Issue: They should use the APIs exported from the icu_testdata crate, like `icu_testdata::get_provider()`.

(Note that this is the API that embeds the absolute path to the data file)

Reply:
I think the point of this example is to demonstrate data loading in general; whether it be from test data or from personally generated data. To me, the testdata here is just one example of a data source that's convenient to use.

Reply:
Ok. However, I think the risk of people thinking that this is the right way to use testdata, and putting these paths into their test file, is greater than the benefit of teaching them about this API. We could link to the FsDataProvider docs which has these examples.

Reply:
I feel like there _is_ no "right way" to use testdata; the crate exists for our internal test uses, this is just for exploratory purposes. I can tighten up the language about this.

Reply:
> I feel like there is no "right way" to use testdata

I disagree. `icu_testdata` is a perfectly fine crate to use if you understand its limitations, and it has public APIs to help using it.

https://docs.rs/icu_testdata/0.2.0/icu_testdata/